### PR TITLE
html and css Files added

### PR DESCRIPTION
**Pull request by brahimdidi**
Created the header and the presentation section of the portfolio mobile version.

### CODE REVIEW

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi **@brahimdidi** , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not meet the professional repository requirements to be reviewed. 
- [ ] The project currently does not have a professional README file. Please checkout this [article](https://microverse.zendesk.com/hc/en-us/articles/360055660814-Professional-GitHub-repository) on how to go about it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limi

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
```markdown
Hi Didi Didi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README. Please add a professional README as was asked by the previous reviewer!

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

Invalid Code Review Request **does not count** into the code reviews limit._
```

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @brahimdidi, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <title>Brahim's Portfolio Page</title>
+    <!-- css links -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+</head>
+<body>
+   <header id="header">
+       <h1 class="headertitle">Jhon Doe</h1>
```
Comment (mke2111) [inline index.html]: - [ ] From the design, this is considered to be the logo. Enclose the logo within `a` tags.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<nav>
+    <img src="template5Images/Union.svg" alt="button">
+</nav>
+   </header>
+   <main class="main">
+  <section class="section1">
+  <div class="Names">
+      <h2 class="h2">Brahim Didi</h2>
```
Comment (mke2111) [inline index.html]: - [ ] This being the main title, change it to a `h1` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      <li class="skills"><h4>Skills</h4></li>
+      <li class="skills">Codekit</li>
+      <li class="skills">Github</li>
+      <li class="skills">Codepen </li>
+      <li class="skills">Gitlab</li>
+      <li class="skills">Terminal</li>
+  </ul>
+  </div>
+  <div class="socials">
+      <ul class="sociallist">
```
Comment (mke2111) [inline index.html:62]: - [ ] So far for your project, I recommend you stick to the font and colors (background and font) defined for the template. Your page has a different background color and fonts.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Brahim's Portfolio Page</title>
+    <!-- css links -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+</head>
+<body>
+   <header id="header">
```
Comment (mke2111) [inline index.html:16]: - [ ] The header is skewed a little to the top, add some margin/padding at the top to match the alignment indicated in the design template

File: index.html (hunk 1 above)
Comment (brahimdidi) [reply index.html]: It does not matter as in this template there is no logo

File: index.html (hunk 2 above)
Comment (brahimdidi) [reply index.html]: those are easy changes that wil require me to make a new request that's really not nice from you

File: index.html (hunk 3 above)
Comment (brahimdidi) [reply index.html:62]: No that is completely wrong I took all the font and colors form the figma template, and they did not  say it should  be exactly the same.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🎉**
Hello @brahimdidi 👋🏽
Your project is complete!  There is nothing else to say but remember in future projects try as much as possible to write a more descriptive pull request. It's time to merge it. 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**